_data/matrix/Bitkit.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bitkit"
+category: "Software Wallet (L1&L2)"
+keyfeatures: "Your ultimate Bitcoin toolkit. Self-custodial BTC and LN payments. Widgets, pay-to-contacts and more"
+homepage: "https://bitkit.to/"
+
+feature:
+  device: "Mobile"
+  os: "Android, iOS"
+  web: "Not Applicable"
```

Comment:
Why is this "Not Applicable" while all other software wallets are "No Support"?

Reply:
Agreed, this should be changed.  I will ensure more clarity as to the meaning of "Not Applicable" & "No Support" so that entry submitters can be more certain about what they submit.